apps/holidays-viewer/templates/add.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+{% block content %}
+  <div class="instructions m-3">
+    <p>Add a new test holiday
+  </div>
+  <div class="base rounded shadow bg-white m-3">
+    <form class="m-auto" action="" method="POST">
+      <div class="form-group">
+        <div class="form-field">
+          <div class="label-field">Date( [Month Day] e.g. January 1 )</div>
+          <input class="bg-secondary border-0 mb-5" name="date">
+        </div>
+        <div class="form-field">
+          <div class="label-field">Description</div>
+          <input class="bg-secondary border-0 mb-5" name="description">
```

Comment:
I'm guessing that this field accepts Wikitext input as well? In any case, it would be great if we add in a placeholder that consists of an example of what an ideal holiday description might look like (be it in plain or wikitext format).